src/openaq/incremental_downloader_parallel.py

```
@@ -1,30 +1,36 @@
+
         # Check if we have parallel client
         self.is_parallel = isinstance(getattr(client.api, '__class__', None).__name__, str) and \
                           'Parallel' in client.api.__class__.__name__
-        
+
         if self.is_parallel:
             print("Using PARALLEL downloader for maximum speed!")
-    
+            print(f"Parallel API client detected with {getattr(client.api, 'num_keys', 'unknown')} keys")
+
+    def _get_sequential_downloader(self):
```

Ruff (0.12.2)

Missing return type annotation for private function `_get_sequential_downloader`

(ANN202)